core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -55,6 +55,10 @@ var ExplorationEditorTranslationTab = function() {
       element(by.cssContainingText('option', language)).click();
   };
 
+  var FeedbackButton = element(
```

Comment:
use lowerCamelCasing here.

Reply:
can we make this feedbackTabButton?

Reply:
Just realised there was already a variable for the same purpose in the file named feedbackTabButton.
Made the change.
done
Thanks